src/Maestro/Microsoft.DotNet.Maestro.Tasks/src/PushMetadataToBuildAssetRegistry.cs

```diff
@@ -494,6 +494,7 @@ internal static (List<PackageArtifactModel>, List<BlobArtifactModel>) GetPackage
                     Attributes = new Dictionary<string, string>
                     {
                         { "NonShipping", package.NonShipping.ToString().ToLower() },
+                        { "NetCoreAsset", package.NetCoreAsset.ToString().ToLower() }
```

Comment:
I am thinking whether these two shouldn't be a constant somewhere. If it's in the here, the model, the XML, the JSON etc

Reply:
I added them to `PushMetadataToBuildAssetRegistry` - not sure it's the best place but it didn't seem right it in the models